packages/neos-ui-redux-store/src/UI/Inspector/index.ts

```diff
@@ -41,6 +41,7 @@ export enum actionTypes {
     DISCARD = '@neos/neos-ui/UI/Inspector/DISCARD',
     ESCAPE = '@neos/neos-ui/UI/Inspector/ESCAPE',
     RESUME = '@neos/neos-ui/UI/Inspector/RESUME',
+    ELEMENT_WAS_SELECTED = '@neos/neos-ui/UI/Inspector/ELEMENT_WAS_SELECTED',
```

Comment:
im not sure that i would get without having this image in mind


![](https://user-images.githubusercontent.com/2522299/239956734-db93bb3b-1624-4f6a-95fb-5043ee94e597.png)


that we are talking about the little dropdown up top ^^ - its easily forgettable. But then again the naming is scoped to the inspector so i think its still fine.

ill try to think of another name ;)